### PR TITLE
section 2: introduction to working with MRI Data in Python

### PR DESCRIPTION
**Pull request by drcandacemakedamoore**
This is the MRI episode, with one tweak to the previous (intro to images) episode.

### CODE REVIEW

**Review by gcroci2 — CHANGES_REQUESTED** [review]
Great lesson :)

I added some suggestions and fixed some typos in https://github.com/esciencecenter-digital-skills/medical-image-processing/pull/11/commits/cdf54db4918a851dc7f9d3fc6006d704f499a844, feel free to revert it or editing the file further. 

A note on the images: I don't see any in the fig/ folder (t1t2flairbrain.jpg, bold.gif, etc.), so I am not sure how they were able to be rendered on my local machine, but they did. I guess that they should be uploaded in the repository for being rendered on the online website, right? Same for the images in the first lesson (introduction_to_medical_imaging.md). If you look carefully indeed, you can see in "Files changes" that the bot gives many missing file warnings in both episodes.

**Comment by drcandacemakedamoore** [comment]
I accept your changes. I am going to add one more change in an upcoming commit based on new conversations with a neuroimaging specialist. Please make sure you accept or request changes on this as well, @gcroci2  then it should be good to approve, I hope.

**Comment by drcandacemakedamoore** [comment]
@gcroci2 your final changes are fine. I'm not sure what happened to the files, so I just added them back in, but also the new ones for your SITK episode- so it's all there now, or should be. If you want to pull off the SITK ones, go ahead, but otherwise, this should now be good to get approved, rebase and merge, I think.

**Comment by gcroci2** [comment]
> @gcroci2 your final changes are fine. I'm not sure what happened to the files, so I just added them back in, but also the new ones for your SITK episode- so it's all there now, or should be. If you want to pull off the SITK ones, go ahead, but otherwise, this should now be good to get approved, rebase and merge, I think.

Please remove the SITK ones, they are in my PR, no need to have them here, then I'll do a last check and we can merge :) @drcandacemakedamoore

**Comment by drcandacemakedamoore** [comment]
@gcroci2 I pulled off the images, but I'm highly pedantic and OCD about using rebase and merge over a plain merge, so unless it bothers you I will rebase and merge.

**Comment by gcroci2** [comment]
All images almost there :) these still missing from episodes/fig folder: `fmri_timeseries.png`, `numpy_arrays.png`, `mri_slices.jpg` @drcandacemakedamoore tag me again when you've added them, I will double check to be sure and then I'll finally approve!

**Comment by drcandacemakedamoore** [comment]
@gcroci2 They are there, I see them in the diff files

**Review by gcroci2 — APPROVED** [review]

**Comment by gcroci2** [comment]
> @gcroci2 They are there, I see them in the diff files

My bad, just approved :)

**Comment by gcroci2** [comment]
Why did you add here all the sitk PR's commits? @drcandacemakedamoore

File: episodes/simpleitk.md [hunk 1]
```diff
@@ -935,6 +935,10 @@ TBD
 
 This episode was largely inspired by [the official SITK tutorial](https://simpleitk.org/TUTORIAL/#tutorial), which is copyrighted by NumFOCUS and distributed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
 
+## Acknowledgements
```
Comment (gcroci2) [inline episodes/simpleitk.md]: This is repeated twice, not sure what happened but I see you added this section @drcandacemakedamoore

**Comment by drcandacemakedamoore** [comment]
Because I rebased on the SITK branch, as they had conflicts ( the config
file and old SITK episode).

**Comment by gcroci2** [comment]
> Because I rebased on the SITK branch, as they had conflicts ( the config file and old SITK episode).

But the SITK branch was already merged into main, so I don't get why this was actually needed. Maybe next time let's do the rebase and merge together, to be sure the flow is clear to both of us (now it's not to me) @drcandacemakedamoore

**Pull request merged**